trusted-types/event-handler-content-attributes.tentative.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title></title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <meta http-equiv="Content-Security-Policy" content="require-trusted-types-for 'script';">
+</head>
+<body>
+<script>
+'use strict';
+[
+  ['div', 'onabort'],
+  ['div', 'onauxclick'],
+  ['body', 'onafterprint'],
```

Comment:
The ordering here is a little weird. Did you consider just sorting alphabetically (resulting in element first, then attribute)?